content/questions/foundations/question-109.md

```diff
@@ -0,0 +1,18 @@
+---
+archetype: "questions"
+title: "Question 109"
+question: "What are the different forms of two-factor or multi-factor authentication supported by GitHub?"
+draft: false
+---
+
+> https://docs.github.com/en/authentication/securing-your-account-with-two-factor-authentication-2fa/accessing-github-using-two-factor-authentication
+- [x] Passkey
+- [x] Text message
+- [ ] Phone call
+> While text messages are supported in some countries, there is no mention of phone calls for 2FA/MFA.
+- [ ] Email
+> There is no mention of email as an option to receive a code for MFA/2FA.
+- [x] Security key
+- [x] GitHub mobile
+- [ ] Biometrics
```

Comment:
@JBonifacio Personally, I would remove it, because while GitHub does not directly use Biometrics, the passkeys (in the browser), it gets used high level with Security keys:

```
Select the appropriate option in the prompt. Depending on your security key configuration, you may type a PIN, complete a biometric prompt, or use a physical security key.
```

Also, TOTP is supported, but not included in the prompts here. It may be a good idea to add that.